pyproject.toml

```diff
@@ -8,7 +8,7 @@ build-backend = "setuptools.build_meta"
 [project]
 name = "ops-scenario"
 
-version = "5.6"
+version = "5.6.1"
```

Comment:
It feels odd to remove functionality in a patch release but maybe these were not considered part of scenario officially? Or maybe you're not so concerned with the numbers at the moment? Anyway, I assume you know best here, just adding a note :)

Reply:
yeah good point. It's not official API, but it's technically non backwards-compatible so it'd warrant a major version bump, still it feels odd for non-official API. Will bump the minor instead.